src/components/withConfirmationPopup/withConfirmationPopup.tsx

```diff
@@ -33,14 +47,18 @@ const withConfirmationPopup = <T extends object>(
 };
 
 interface PopupContentProps {
+  deleteType: string;
+  deleteName: string;
   setOpen: Function;
   onConfirm: () => void;
 }
 
-const PopupContent: FC<PopupContentProps> = ({ setOpen, onConfirm }) => {
+const PopupContent: FC<PopupContentProps> = ({ deleteType, deleteName, setOpen, onConfirm }) => {
```

Comment:
The text is nice but now this component is specific to deletion. Could we make this just one single text parameter, which is optional, that gets displayed in the popup? If not provided it would default to just "Are you sure" as before.